README.md

```diff
@@ -1 +1,133 @@
-# to-do-list
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+- [👤 Author](#author)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 To Do List App <a name="about-project"></a>
+
+'o Do List' is the project of pure js in the Microverse program 2nd module, made for learning purposes.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Languages</summary>
+  <ul>
+    <li>HMTL</li>
+    <li>CSS</li>
+    <li>JavaScript</li>
+  </ul>
+</details>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Key Features <a name="key-features"></a>
+
+- **Dynamically generated HTML**
+- **Responsive for both desktop and mobile versions**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+You can checkout the running website here:
+
+- [Live Demo Link](https://github.com/JakariaJishan/to-do-list/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps:
+
+### Prerequisites
+
+In order to run this project you need:
+
+- VS Code installed.
+- A GitHub account.
+- Git Bash.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- On VS Code, you can click on the "Source Control" icon on the left side bar.
+- From there click on the "Clone Repository" button.
+- Paste the URL from this project and choose a folder to store it.
+- Open the project.
+
+I do recommend this method given it is easier, but if you want you can make this process by running commands on the Git Bash terminal.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHOR -->
+
+## 👤 Author <a name="author"></a>
+
+Jakaria Jishan
+
+- GitHub: [@Jakaria jishan](https://github.com/JakariaJishan)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **Additional functional pages**
+- **User info collection through forms**
+
```

Comment:
- [ ] For this section, kindly add some of the future features you will add such as `add and remove` books, `renaming list items` e.t.c. as this will make your Readme more descriptive